x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_related_detail.tsx

```diff
@@ -0,0 +1,342 @@
+        type: 'userRequestedRelatedEventData',
+        payload: processEntityId,
+      });
+    }
+  }, [relatedsReady, dispatch, processEntityId]);
+
+  const relatedEventsForThisProcess = useSelector(selectors.relatedEventsByEntityId).get(
+    processEntityId!
+  );
+
+  const [relatedEventToShowDetailsFor, countBySameCategory, relatedEventCategory] = useMemo(() => {
+    if (!relatedEventsForThisProcess) {
+      return [undefined, 0];
+    }
+    const specificEvent = relatedEventsForThisProcess.events.find(
+      (evt) => event.eventId(evt) === relatedEventId
+    );
+    // For breadcrumbs:
+    const specificCategory = specificEvent && event.eventCategory(specificEvent);
+    const countOfCategory = relatedEventsForThisProcess.events.reduce((sumtotal, evt) => {
+      return event.eventCategory(evt) === specificCategory ? sumtotal + 1 : sumtotal;
+    }, 0);
+    return [specificEvent, countOfCategory, specificCategory || naString];
+  }, [relatedEventsForThisProcess, naString, relatedEventId]);
+
+  const [sections, formattedDate] = useMemo(() => {
+    if (!relatedEventToShowDetailsFor) {
+      // This could happen if user relaods from URL param and requests an eventId that no longer exists
+      return [[], naString];
+    }
+    // Assuming these details (agent, ecs, process) aren't as helpful, can revisit
+    const {
+      agent,
+      ecs,
+      process,
+      ...relevantData
+    } = relatedEventToShowDetailsFor as ResolverEvent & {
+      ecs: unknown;
+    };
+    let displayDate = '';
+    const sectionData: Array<{
+      sectionTitle: string;
+      entries: Array<{ title: string; description: string }>;
+    }> = Object.entries(relevantData)
+      .map(([sectionTitle, val]) => {
+        if (sectionTitle === '@timestamp') {
+          displayDate = formatDate(val);
+          return { sectionTitle: '', entries: [] };
+        }
+        if (typeof val !== 'object') {
+          return { sectionTitle, entries: [{ title: sectionTitle, description: `${val}` }] };
+        }
+        return { sectionTitle, entries: [...objectToDescriptionListEntries(val)] };
+      })
+      .filter((v) => v.sectionTitle !== '' && v.entries.length);
+    return [sectionData, displayDate];
+  }, [relatedEventToShowDetailsFor, naString]);
+
+  const waitCrumbs = useMemo(() => {
+    return [
+      {
+        text: eventsString,
+        onClick: () => {
+          pushToQueryParams({ crumbId: '', crumbEvent: '' });
+        },
+      },
+    ];
+  }, [pushToQueryParams, eventsString]);
+
+  const crumbs = useMemo(() => {
+    return [
+      {
+        text: eventsString,
+        onClick: () => {
+          pushToQueryParams({ crumbId: '', crumbEvent: '' });
+        },
+      },
+      {
+        text: processName,
+        onClick: () => {
+          pushToQueryParams({ crumbId: processEntityId!, crumbEvent: '' });
+        },
+      },
+      {
+        text: (
+          <>
+            <EuiI18nNumber value={totalCount} />

```

Comment:
It seems like this should i18n'd. like 
```
<FormattedMessage
  id="nerds"
  defaultMessage="{totalCount} Events"
  values={{ totalCount }}
  />
```
This will allow the translator flexibility in how they present the number and label

Reply:
Gotcha here: https://github.com/elastic/kibana/pull/69098/commits/ee1f0e69e885e625bc62179d059a58d6116334f1